stream/publisher_test.go

```diff
@@ -173,6 +173,9 @@ func (s *publisherTestSuite) TestCannotResolveTopicID() {
 }
 
 func (s *publisherTestSuite) TestPublisherTopicIDs() {
+	// TODO: create a story to fix this test
+	s.T().Skip("this test is causing failures in CI")
+
```

Comment:
Added [SC-20183](https://app.shortcut.com/rotational/story/20183/fix-testpublishertopicids-timeout) for this TODO